client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -129,6 +135,9 @@ class TransferDomainPrecheck extends React.Component {
 		} else if ( false === unlocked ) {
 			heading = translate( 'Unlock the domain.' );
 		}
+		if ( loading ) {
```

Comment:
Interesting that we missed those. These should be `loading && ! isStepFinished`, because if we're on the second step, the copy shouldn't change. Check line 209.